File: src/targets/node/unirest/fixtures/query.js
const unirest = require('unirest');

<<<<<<< HEAD
const req = unirest('GET', 'https://httpbin.org/anything');
=======
const req = unirest('GET', 'http://mockbin.com/har');
>>>>>>> upstream/master

req.query({
  foo: [
    'bar',
    'baz'
  ],
  baz: 'abc',
  key: 'value'
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/targets/java/unirest/fixtures/query.java
<<<<<<< HEAD
HttpResponse<String> response = Unirest.get("https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value")
=======
HttpResponse<String> response = Unirest.get("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value")
>>>>>>> upstream/master
  .asString();